frontend/public/components/utils/_resource-log.scss

```diff
@@ -0,0 +1,13 @@
+.resource-log {
+  height: 100%;
+  width: 100%;
+  &.log-console--fullscreen {
+    .co-toolbar {
+      padding-left: 10px;
+      padding-right: 10px;
+    }
+  }
+  .pairs-list__action-divider {
+    padding: 0;
```

Comment:
This is not the pairs-list block. We either need to generalize this class or have a special class for the log viewer

Reply:
fixed, third time's a charm :-)

Reply:
You could probably make this a part of the `co-toolbar` styles in `frontend/styles/_common.scss`. Something like `.co-toolbar__item--divider`.